src/factory/providers/zksync-era/fulcrom/data.json

```diff
@@ -0,0 +1,7 @@
+{
+  "address": "0x7d5b0215EF203D0660BC37d5D09d964fd6b55a1E",
```

Comment:
Can you make sure that the FUL address is correct?
It looks different at Coingecko.
[0xe593853b4d603d5b8f21036bb4ad0d1880097a6e](https://www.coingecko.com/en/coins/fulcrom)

Reply:
Thanks for pointing out, here should be the contract of FUL address in zkSync, which is `0xe593853b4d603d5b8f21036Bb4AD0D1880097a6e`, changed commited @Sonmezturk